hibernate-core/src/main/java/org/hibernate/jpa/internal/MutableJpaComplianceImpl.java

```diff
@@ -92,6 +93,12 @@ public MutableJpaComplianceImpl(Map configurationSettings, boolean jpaByDefault)
 				configurationSettings,
 				jpaByDefault
 		);
+
+		criteriaCopyCompliance = ConfigurationHelper.getBoolean(
+				AvailableSettings.JPA_CRITERIA_COPY_COMPLIANCE,
+				configurationSettings,
+				true
```

Comment:
Right, here is the crux of my comment about the default.  All of the others default to `jpaByDefault`.  I believe this should be the same

Reply:
Hmm, if you think this the right thing to do, then I'll do it. I'll direct the users with complaints about this to you then 😜